src/modules/users/user_repository.ts

```diff
@@ -39,4 +39,6 @@ export namespace User {
   }
 
   export const findById = async (id: string) => Query().where('users.id', id).first()
+
+  export const destroy = async (id: number) => Users().where('id', id).delete()
```

Comment:
it seems that this function doesn't need to be async?
`destroy` already and only returns promise, u just need to call the function from the service with `await`

Reply:
yes thank for your information, next we will remove async when return promise

Reply:
ngokhey, but it's worth pointing out, this only applies if in a function block there is only one promise that doesn't depend on each other